packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -182,7 +183,10 @@ class CallContext {
         page: newPage,
         endedCall: latestEndedCall,
         devices: clientState.deviceManager,
-        latestErrors: clientState.latestErrors
+        latestErrors: clientState.latestErrors,
+        isCameraOn:
+          clientState.deviceManager.unparentedViews.find((s) => s.mediaStreamType === 'Video') !== undefined ||
```

Comment:
I was going to comment that nit: `!== undefined` is a double negative, consider an is defined instead
```suggestion
          !!clientState.deviceManager.unparentedViews.find((s) => s.mediaStreamType === 'Video') ||
```

But turns out that check is literally a double negative (`!!`) check in itself 😄